### PR TITLE
Add `woocommerce-github-sync-labels` config.

### PR DESCRIPTION
**Pull request by tomalec**
### Changes proposed in this Pull Request:

- Add `woocommerce-github-sync-labels` config.
   So we could easily add labels used by https://github.com/woocommerce/grow/pull/38 to all our repos and somewhat maintain that list in future.

-  Fix `changelog: doc` label in `release.yml`

### Detailed test instructions:

1. Follow instruction steps from the README, up to `dry-run`
2. Running actual command, could add labels to the repos, that may require manual delete.

### Changelog entry

> Add - `woocommerce-github-sync-labels` config.

### CODE REVIEW

File: configs/woocommerce-github-sync-labels/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+This folder contains configs to be used with [`woocommerce-github-sync-labels`](https://github.com/woocommerce/woocommerce-github-sync-labels) to automatically, sync those labels with all our repos.
+
+## How to use
+
+1. [Install and setup `woocommerce-github-sync-labels`](https://github.com/woocommerce/woocommerce-github-sync-labels#install)
+2. Copy configs from this folder into the cloned tool folder 
+3. Run dry run
+	```
+	npm start -- --dry-run --preserve-labels
+	```
```
Comment (puntope) [inline configs/woocommerce-github-sync-labels/README.md:14]: ❓ Just curious, why we need to add `--dry-run` with `npm start`  ??

File: configs/woocommerce-github-sync-labels/labels/commons.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "name": "changelog: tweak",
+    "color": "5c7588",
+    "description": "Small change, that isn't actually very important."
+  },
+  {
+    "name": "changelog: update",
+    "color": "5c7588",
+    "description": "Big changes to something that wasn't broken."
+  }
+]
```
Comment (puntope) [inline configs/woocommerce-github-sync-labels/labels/commons.json]: 💅  Missing newline at the end of the file

Screenshot 2022-08-02 at 13 47 37

File: configs/woocommerce-github-sync-labels/repositories.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "woocommerce/woocommerce-coupon-campaigns",
+  "woocommerce/woocommerce-drip",
+  "woocommerce/woocommerce-follow-up-emails",
+  "woocommerce/woocommerce-give-products",
+  "woocommerce/woocommerce-google-analytics-integration",
+  "woocommerce/woocommerce-points-and-rewards",
+  "woocommerce/woocommerce-product-filters",
+  "woocommerce/woocommerce-product-finder",
+  "woocommerce/woocommerce-sales-report-email"
+]
```
Comment (puntope) [inline configs/woocommerce-github-sync-labels/repositories.json]: 💅 Missing newline at the end of the file

File: configs/woocommerce-github-sync-labels/README.md (hunk 1 above)
Comment (puntope) [reply configs/woocommerce-github-sync-labels/README.md:14]: Ok, I can see in https://github.com/woocommerce/woocommerce-github-sync-labels#install.

This is for testing? I think `npm test -- --preserve-labels` also works?

**Review by puntope — APPROVED** [review]
✅  LGTM

💅  Some issues to be addressed in regards EOF newlines. But approved in advance
 
- Tested locally with dry-run

```
➜  woocommerce-github-sync-labels git:(master) ✗ npm test --  --preserve-labels

> woocommerce-github-sync-labels@0.0.3 test
> node index.js --dry-run "--preserve-labels"


Starting sync...
✔️ "--dry-run" is enabled. No changes will happen in any repository.
✔️ "--preserve-labels" is enabled. New labels introduced directly in GitHub will be preserved.

Repositories:
  -woocommerce/automatewoo
  -woocommerce/automatewoo-birthdays
  -woocommerce/automatewoo-referrals
  -woocommerce/facebook-for-woocommerce
  -woocommerce/google-listings-and-ads
  -woocommerce/pinterest-for-woocommerce
  -woocommerce/woocommerce-brands
  -woocommerce/woocommerce-coupon-campaigns
  -woocommerce/woocommerce-drip
  -woocommerce/woocommerce-follow-up-emails
  -woocommerce/woocommerce-give-products
  -woocommerce/woocommerce-google-analytics-integration
  -woocommerce/woocommerce-points-and-rewards
  -woocommerce/woocommerce-product-filters
  -woocommerce/woocommerce-product-finder
  -woocommerce/woocommerce-sales-report-email
Labels synced for "woocommerce/woocommerce-brands"
Labels synced for "woocommerce/woocommerce-sales-report-email"
Labels synced for "woocommerce/woocommerce-google-analytics-integration"
Labels synced for "woocommerce/automatewoo"
Labels synced for "woocommerce/automatewoo-referrals"
Labels synced for "woocommerce/woocommerce-drip"
Labels synced for "woocommerce/woocommerce-product-filters"
Labels synced for "woocommerce/facebook-for-woocommerce"
Labels synced for "woocommerce/woocommerce-give-products"
Labels synced for "woocommerce/woocommerce-product-finder"
Labels synced for "woocommerce/woocommerce-points-and-rewards"
Labels synced for "woocommerce/pinterest-for-woocommerce"
Labels synced for "woocommerce/automatewoo-birthdays"
Labels synced for "woocommerce/woocommerce-coupon-campaigns"
Labels synced for "woocommerce/woocommerce-follow-up-emails"
Labels synced for "woocommerce/google-listings-and-ads"
```

File: configs/woocommerce-github-sync-labels/README.md (hunk 1 above)
Comment (tomalec) [reply configs/woocommerce-github-sync-labels/README.md:14]: Yes, this is for testing. Given the operation is not easily reversible, I think the dry run/simulation is valuable.

**Comment by tomalec** [comment]
@puntope I added your fix from https://github.com/woocommerce/pinterest-for-woocommerce/pull/505#discussion_r937611567 to here as well, to keep the label names in sync between release and labels configs.

**Review by puntope — APPROVED** [review]
Thanks for the tweak! LGTM

**Pull request merged**